programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md

```diff
@@ -101,6 +102,40 @@ Test coverage could increase
 - Mentor(s): Reto Lehmann @ReToCode (rlehmann AT redhat DOT com),  Stavros Kontopoulos @skonto (skontopo AT redhat DOT com)
 - Upstream Issue (URL): https://github.com/knative/serving/issues/12718
 
+### Kubescape
+
+#### Store Kubescape configuration scan results as CRs
+
+- Description: Implement a mechanism in the Kubescape operator which detects changes in the cluster and triggers a scan of the cluster.
```

Comment:
Could you add a bit of more description here @craigbox ? Why this is beneficial? What are the current problems?

Reply:
Done